cli/commands/quote.ts

```diff
@@ -84,13 +90,14 @@ export class Quote extends BaseCommand {
       tokenOutStr,
     ]);
 
+    // if the tokenIn str is 'ETH' or 'MATIC' or NATIVE_CURRENCY_STRING
     const tokenIn: Currency =
-      tokenInStr == 'ETH'
-        ? Ether.onChain(chainId)
+      tokenInStr in NativeCurrencyName
```

Comment:
fyi, the interface just treats MATIC as 'ETH' atm

Reply:
ok, this is fine cause its just CLI and the rest should be fine with the interface since you're just calling route